lib/e2e/Cargo.toml

```
@@ -2,10 +2,11 @@
 name = "e2e"
 description = "End-to-end Testing for Stylus"
 version = "0.1.0"
+categories = ["development-tools::testing", "cryptography::cryptocurrencies"]
+keywords = ["arbitrum", "ethereum", "stylus", "tests", "openzeppelin"]
```

We're not including OZ as a keyword in the other packages, and I think is not the best practice to include company name(and it is already part of the package name and owners)
```suggestion
keywords = ["arbitrum", "ethereum", "stylus", "tests"]
```